concepts/modularization/README.md

```diff
@@ -0,0 +1,83 @@
+# Modularization
```

Comment:
I propose to include already on this level as a requirement a unified adoption of microdelivery process , it should be included in module governance. 

What I'd like to achieve is to keep consistency across all modules in terms of adoption of one microdelivery process for all modules that also result in:
- having same branching strategy
- handling vulnerabilities 
- same approach for maintenance releases (including security fixes)

All together it will let us have unified support model (together with the operational awareness introduced by @tobiscr).

Having freedom in adoption of microdelivery process could result in a significant effort needed later for release management and support (unification will not be possible on that level).  CC @Sawthis

Reply:
CC: @kasiakepka @dekiel